packages/mobile/src/transactions/TransactionsList.tsx

```diff
@@ -0,0 +1,310 @@
+import BigNumber from 'bignumber.js'
+import gql from 'graphql-tag'
+import * as React from 'react'
+import { Query } from 'react-apollo'
+import { connect } from 'react-redux'
+import {
+  MoneyAmount,
+  Token,
+  TransactionType,
+  UserTransactionsQuery,
+  UserTransactionsQueryVariables,
+} from 'src/apollo/types'
+import { CURRENCIES, CURRENCY_ENUM } from 'src/geth/consts'
+import { SENTINEL_INVITE_COMMENT } from 'src/invite/actions'
+import { LocalCurrencyCode } from 'src/localCurrency/consts'
+import { getLocalCurrencyCode, getLocalCurrencyExchangeRate } from 'src/localCurrency/selectors'
+import { RootState } from 'src/redux/reducers'
+import { removeStandbyTransaction } from 'src/transactions/actions'
+import {
+  ExchangeStandby,
+  StandbyTransaction,
+  TransactionStatus,
+  TransferStandby,
+} from 'src/transactions/reducer'
+import TransactionFeed, { FeedItem, FeedType } from 'src/transactions/TransactionFeed'
+import { currentAccountSelector } from 'src/web3/selectors'
+
+// Query poll interval
+const POLL_INTERVAL = 10000 // 10 secs
+
+interface OwnProps {
+  currency: CURRENCY_ENUM
+}
+
+interface StateProps {
+  address?: string | null
+  standbyTransactions: StandbyTransaction[]
+  localCurrencyCode: LocalCurrencyCode | null
+  localCurrencyExchangeRate: string | null | undefined
+}
+
+interface DispatchProps {
+  removeStandbyTransaction: typeof removeStandbyTransaction
+}
+
+type Props = OwnProps & StateProps & DispatchProps
+
+// See https://github.com/microsoft/TypeScript/issues/16069#issuecomment-565658443
+function isPresent<T>(t: T | undefined | null | void): t is T {
```

Comment:
Consider moving this to a shared place. Like utils/typescript.ts

Reply:
Yep!